pool_automation/roles/aws_manage/defaults/main.yml

```diff
@@ -23,15 +25,19 @@ aws_regions:
   - us-west-1
   - us-west-2
 
-instance_count: 4
-
-pa_aws_prefix: PA
-
-# Derivative parameters
-group_name: "{{ aws_tag_role }}s"
+#   EC2 instance type
+aws_ec2_type: t2.micro
 
+#   Resource tags and names
+aws_tag_project: "{{ aws_project_name }}"
 aws_tag_namespace: "{{ inventory_dir | default('test', true) | basename }}"
+aws_tag_role: default
+aws_add_tags: {} # additional tags
+aws_keyname: "{{ [aws_project_name_short, aws_tag_namespace, group_name]|join('-')|lower }}"
+aws_sgroup: "{{ [aws_project_name_short, aws_tag_namespace, group_name]|join('-')|lower }}"
```

Comment:
Looks like a candidate for extraction to common var

Reply:
yes,exactly. Fixed